File: Hello SolutionDenisT/FisrtDigit/DigitiListener.cs
using System;

namespace FisrtDigit {
    public class DigitiListener {
        public virtual int EnterDigit() {
            Console.WriteLine("Enter 1st number>25");
            var number = Console.ReadLine();
            if (number != null) {
                int result;
                var isPrsed = int.TryParse(number, out result);
                if (isPrsed) {
                    return result;
                }
            }
            return 0;
        }
    }
}

File: Hello SolutionDenisT/ThirdDigit/DigitiListener.vb
Public Class DigitiListener
    Inherits FisrtDigit.DigitiListener

    Public Overrides Function EnterDigit() As Integer
        Console.WriteLine("Enter 3rd number>3")
        Dim number As String
        number = Console.ReadLine()
        Return number.ToString()
    End Function
End Class
